<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetLibraryView</class>
 <widget class="QMainWindow" name="WidgetLibraryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="toolFrameLibraryStatus">
      <property name="styleSheet">
       <string notr="true">QToolButton {
    border: 1px solid transparent;
    background-color: transparent;
}

 QToolButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

QToolButton:checked {
     border: 1px solid rgb(78, 96, 255);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
}

 QToolButton:hover {
     border: 1px solid rgb(78, 96, 255);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QToolButton[popupMode=&quot;1&quot;] { /* only for MenuButtonPopup */
	padding-left: -4px;
    padding-right: 20px; /* make way for the popup button */
 }

/* the subcontrols below are used only in the MenuButtonPopup mode */
 QToolButton::menu-button {
     border: 1px solid transparent;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

QToolButton::menu-button:hover {
    border: 1px solid rgb(78, 96, 255);
	border-left-color: 1px solid transparent;
     /* 16px width + 4px for border = 20px allocated above */
     width: 16px;
 }

 QToolButton::menu-arrow:open {
     top: 1px; left: 1px; /* shift it a bit */
 }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="horizontalSpacing">
        <number>1</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLabel" name="labelLibraryStatus">
         <property name="text">
          <string>Exploring Shared Folders</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelLibraryStatusLogo">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../Resource.qrc">:/Resource/Library/SharedFolders.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="toolButtonLibraryViewType">
         <property name="text">
          <string>Tile View</string>
         </property>
         <property name="icon">
          <iconset resource="../Resource.qrc">
           <normaloff>:/Resource/Library/TileView.png</normaloff>:/Resource/Library/TileView.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::MenuButtonPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacerLibraryStatus">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitterLibraryView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeView" name="treeViewLibraryFiles">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
      </widget>
      <widget class="QFrame" name="frameLibraryFileDetails">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollAreaLibraryFileDetails">
          <property name="styleSheet">
           <string notr="true">QFrame {
	background-color: white;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QFrame" name="frameLibraryFileDetailsContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>558</width>
             <height>82</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_11">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLibraryAddAlbum"/>
   <addaction name="actionLibraryDeleteAlbum"/>
  </widget>
  <widget class="QToolBar" name="toolBarSearch">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFind"/>
  </widget>
  <action name="actionLibraryAddAlbum">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Add.png</normaloff>:/Resource/Generic/Add.png</iconset>
   </property>
   <property name="text">
    <string>Add Album</string>
   </property>
  </action>
  <action name="actionLibraryDeleteAlbum">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Remove.png</normaloff>:/Resource/Generic/Remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete Album</string>
   </property>
   <property name="toolTip">
    <string>Delete Album</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Search.png</normaloff>:/Resource/Generic/Search.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
